### ISSUE TITLE
Trinity: start trinity in a postscript

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#283]
That way we can do some tweaking before the trinity container is actually started in earlier postscripts. E.g. setup storage, setup network interfaces etc

**Comment by hansthen** [comment]
This is implemented in 3236894ab together with the start docker from LVM.

**Comment by hansthen** [comment]
**Version:** r4_fixes cdaf89b
**Environment:** master
```
[root@node001 ~]# docker ps
CONTAINER ID        IMAGE                            COMMAND                CREATED              STATUS              PORTS               NAMES
cf8c77927122        controller:5050/trinity:latest   "/usr/bin/supervisor   About a minute ago   Up About a minute                       trinity 
```
